src/features/chat/model/useChatWebSocketStore.ts

```
@@ -0,0 +1,100 @@
+          rooms: Array.isArray(response) ? response : [],
+        });
+      },
+
+      setRooms: (response) => {
+        switch (response.eventType) {
+          case 'CREATE': {
+            set((state) => {
+              return { rooms: [...state.rooms, response] };
+            });
+          }
+          case 'DELETE': {
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)